Automattic/wp-calypso
# Style: Use `@wordpress/base-styles`, drop local copy

#### Changes proposed in this Pull Request

[`@wordpress/base-styles`](https://github.com/WordPress/gutenberg/tree/3ab776b5862f5f36701d2baa231eab949266372b/packages/base-styles) was published yesterday. Use it instead of our local copy.

#### Testing instructions

Verify that the app still builds and runs.
Verify that `http://calypso.localhost:3000/gutenboarding` still looks the same as before.
